Makefile

```diff
@@ -1,7 +1,8 @@
 GFA_FILES := k.gfa DRB1-3123.gfa
+TEST_FILE := DRB1-3123.gfa
 GFA_URL := https://raw.githubusercontent.com/pangenome/odgi/ebc493f2622f49f1e67c63c1935d68967cd16d85/test
 
-.PHONY: fetch
+.PHONY: fetch test
```

Comment:
Just stating the obvious: looks like these Makefile changes aren't used for now.